src/styles/blocks/boxes.scss

```diff
@@ -0,0 +1,33 @@
+.boxes {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  
+  @media (min-width: 900px) {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  @for $box-num from 1 through 6 {
+    &__#{$box-num}:nth-child(#{$box-num})::after {
+      content: "#{$box-num}";
+      height: 100%;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+  }
+
+  $color-list:
+    1 #f00 "red",
+    2 #c00 'red2',
+    3 #900 'red-black',
+    4 #600 'black-red',
+    5 #300 'black2',
+    6 #000 'black';
```

Comment:
move to variables file

Reply:
I think you can achieve the same by assigning colours to boxes just according to their indexes, this way you can omit using modifier classes